docs/modules/sql/pages/alter-job.adoc

```diff
@@ -24,28 +24,33 @@ The `job_name` parameter is required.
 |===
 |Parameter | Description | Example
 
-|job_name
+|`job_name`
 |The name of the job to suspend, resume, or restart.
 |
 
-|SUSPEND
+|`SUSPEND`
 |Suspend the job. For details, see the API reference for the link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/Job.html#suspend()[`suspend()`] method.
-|
+|<<suspend, Suspend and Resume a Job>>
+
+|`OPTIONS`
+|Applies new configuration options to a suspended job. See xref:pipelines:configuring-jobs.adoc#job-configuration-options[job configuration options] for valid values for the `option_name` parameter. For more details, see the API for the Job interface: link:https://docs.hazelcast.org/docs/{full-version}/javadoc/com/hazelcast/jet/config/DeltaJobConfig.html[`updateConfig(DeltaJobConfig deltaConfig)`].
```

Comment:
The applicable options for `ALTER JOB` statement is a subset of the options for `CREATE JOB` statement. The applicable options should at least be named here.

Reply:
Updated as requested. I noticed that the ``suspendOnFailure` option was not included on the following page: https://docs.hazelcast.com/hazelcast/5.3-snapshot/pipelines/configuring-jobs#latest-banner so I have added this to the PR.